tools/BackPopulateSNRAndAcquisitionOrder.pl

```diff
@@ -52,7 +54,7 @@
 if ($profile && !@Settings::db) {
     print "\n\tERROR: You don't have a configuration file named ".
           "'$profile' in:  $ENV{LORIS_CONFIG}/.loris_mri/ \n\n";
-    exit 2;
+    exit $NeuroDB::ExitCodes::DB_SETTINGS_FAILURE;
```

Comment:
@cmadjar 
did you not decide to split this condition into 2 in your other scripts?
you have done this in all otgher sctipts I think:

```
if ( !$profile ) {
    print $Help;
    print STDERR "$Usage\n\tERROR: missing -profile argument\n\n";
    exit $NeuroDB::ExitCodes::PROFILE_FAILURE;
}
{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
if ( !@Settings::db ) {
    print STDERR "\n\tERROR: You don't have a \@db setting in the file "
                 . "$ENV{LORIS_CONFIG}/.loris_mri/$profile \n\n";
    exit $NeuroDB::ExitCodes::DB_SETTINGS_FAILURE;
}
```